Not in virtualenv
cuDNN not found!
bazel not found
{'label': 'programming_language', 'name': 'python', 'version': '3.8', 'verdict': 'yes', 'post_id': ['68917678']}
{'label': 'programming_language', 'name': 'python', 'version': '3.9', 'verdict': 'yes', 'post_id': ['68917678']}
......................................................................................
Current library: numpy (v1.21.6)
Knowledge base: {'library': {}, 'hardware': {}, 'programming_language': {'python': [('3.8', 'yes', ['68917678']), ('3.9', 'yes', ['68917678'])]}, 'operating_system': {}, 'tool': {}, 'database': {}, 'software': {}}
unknown
unknown
{'label': 'hardware', 'name': 'cuda', 'version': '10.1', 'verdict': 'no', 'post_id': ['56426255']}
{'label': 'operating_system', 'name': 'ubuntu', 'version': '18.04.2', 'verdict': 'yes', 'post_id': ['55188392']}
{'label': 'hardware', 'name': 'cuda', 'version': '9.1', 'verdict': 'yes', 'post_id': ['53345147']}
{'label': 'operating_system', 'name': 'ubuntu', 'version': '16.04', 'verdict': 'unknown', 'post_id': ['53345147']}
{'label': 'hardware', 'name': 'cuda', 'version': '9', 'verdict': 'yes', 'post_id': ['53727997', '50622526', '53596618', '53457472', '55782472', '60077642']}
{'label': 'hardware', 'name': 'cudnn', 'version': '7.1.4', 'verdict': 'yes', 'post_id': ['53727997', '50622526']}
......................................................................................
Current library: tensorflow (v1.12.0)
Knowledge base: {'library': {}, 'hardware': {'cuda': [('10.1', 'no', ['56426255']), ('9.1', 'yes', ['53345147']), ('9', 'yes', ['53727997', '50622526', '53596618', '53457472', '55782472', '60077642'])], 'cudnn': [('7.1.4', 'yes', ['53727997', '50622526'])]}, 'programming_language': {}, 'operating_system': {'ubuntu': [('18.04.2', 'yes', ['55188392']), ('16.04', 'unknown', ['53345147'])]}, 'tool': {}, 'database': {}, 'software': {}}
--------------------------------DC ISSUE-----------------------------------------
ERROR: Local tensorflow (v1.12.0) is not compatible with cuda (v11.7) (0)
-----------------------------------END-------------------------------------------
--------------------------------DC ISSUE-----------------------------------------
cudnn is not installed in local, the recommended version is ['7.1.4']
-----------------------------------END-------------------------------------------
matplotlib is not installed in local
opencv-python is not installed in local
